### PR TITLE
PIM-3674: add view elements to js_handlers

### PR DESCRIPTION
**Pull request by sumbobyboys**
| Q                    | A
| -------------------- | ---
| Bug fix?             | N
| New feature?         | Y
| BC breaks?           | N
| CI currently passes? | Y
| Tests pass?          | Y
| Scenarios pass?      | Y
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | N
| Fixed tickets        | PIM-3674
| DB schema updated?   | N
| Migration script?    | N
| Doc PR               |

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/views/Product/_js-handler.html.twig [hunk 1]
```diff
@@ -92,6 +92,8 @@
                     new DialogForm('#create-product');
                     $('#create-product').click();
                 {% endif %}
+
+                {{ view_elements('pim_enrich_product_edit.after_form_init') }}
```
Comment (filipsalpe) [inline src/Pim/Bundle/EnrichBundle/Resources/views/Product/_js-handler.html.twig]: inside the require? ugly if scripts have more deps

Comment (filipsalpe) [inline src/Pim/Bundle/EnrichBundle/Resources/views/Product/_js-handler.html.twig]: Maybe having these js handler scripts as view elements could be nicer too - that would create the possibility to remove them if total override is needed

Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/views/Product/_js-handler.html.twig]: ok, I will change that

Comment (Nuscly) [inline src/Pim/Bundle/EnrichBundle/Resources/views/Product/_js-handler.html.twig]: for most case, better than overriding all the js

Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/views/Product/_js-handler.html.twig]: We will fix that later, for now this is a first step

Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/views/Product/_js-handler.html.twig]: @Nuscly yes, this is always better, but not nice and clean. We will have to rework that.

**Pull request merged**